apps/discordbot/bot.py

```diff
@@ -55,8 +77,16 @@ async def on_message(self, message):
 
     async def on_member_join(self, member):
         await member.send(
-            "Welcome to the CSUA discord server! Verify your berkeley.edu email to gain access. Please contact a moderator if you have any issues."
+            "Welcome to the CSUA discord server! First, read the rules in #landing-zone. Thumbs up this message if you agree")
+        channel = member.dm_channel
+        def check_thumb(react, user):
+            return user == member and str(react.emoji) == '👍' # thumbs
+        await self.wait_for('reaction_add', check=check_thumb)
```

Comment:
Does this check that they thumbs up that message? Wouldn't it work for any message they thumbs up? I think you can add `message = await member.send(...)`followed by `react.message==message`? Or maybe `react.message.channel == channel`?

Reply:
makes sense, will add

Reply:
fixed